axelrod/strategies/shortmem.py

```diff
@@ -0,0 +1,50 @@
+from axelrod import Actions, Player
+from axelrod.actions import Action
+
+C, D = Actions.C, Actions.D
+
+class ShortMem(Player):
+    """
+    A player starts by always cooperating for the first 10 moves.
+    
+    The opponent answers are stored in the memory, whose maximum size is 10
+    results. From the tenth round on, the program analyzes the memory, and
+    compare the number of defects and cooperates of the opponent, based in
+    percentage. If cooperation occurs 30% more than defection, it will
+    cooperate.
+    If defection occurs 30% more than cooperation, the program will defect.
+    Otherwise, the program follows the TitForTat algorithm.
+    
+    Names:
+    
+    - ShortMem: [Andre2013]_
+    """
+
+    name = 'ShortMem'
+    classifier = {
+        'memory_depth': 10,
+        'stochastic': False,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    @staticmethod
+    def strategy(opponent: Player) -> Action:
+        
+        if len(opponent.history) <= 10:
+            return C
+        
+        array = opponent.history[:-11:-1]
```

Comment:
> Sorry for being difficult, but I really don't see the reason for an additional test.

Don't worry you're not being difficult, it's good to talk these things over :)

I would definitely suggest you change:

```
opponent.history[:-11:-1]
```

to be 

```
opponent.history[-10:]
```

This just helps with readability.

In terms of the test, I suggest you just add the following one:

```
self.responses_test([C], [D] *9, [D]  * 9)  # This checks that when there are 9 previous plays you cooperate
self.responses_test([D], [D] *10, [D]  * 10)  # This checks that when there are 10 previous plays the logic kicks in
```